ckanext/activity/model/activity.py

```diff
@@ -15,6 +15,8 @@
     not_,
     union_all,
     text,
+    select,
+    literal
```

Comment:
It would be better to have a trailing comma here, so that adding more lines (as demonstrated in this change) doesn't require changing an existing line.